### PR TITLE
(issue 179): fixing exception raised when opening session with non array objects

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

File: larray_editor/editor.py [hunk 1]
```diff
@@ -396,11 +396,14 @@ def void_formatter(array, *args, **kwargs):
                 self.new()
         # convert input data to Session if not
         else:
-            self.data = data if isinstance(data, Session) else Session(data)
-            if qtconsole_available:
-                self.kernel.shell.push(dict(self.data.items()))
-            arrays = [k for k, v in self.data.items() if self._display_in_grid(k, v)]
-            self.add_list_items(arrays)
+            self._push_data(data)
```
Comment (gdementen) [inline larray_editor/editor.py]: Is it intended that "self._listwidget.setCurrentRow(0)" is not longer called in the self.new() case above? It is probably fine but have you checked that?

File: larray_editor/editor.py [hunk 2]
```diff
@@ -933,8 +936,7 @@ def _open_file(self, filepath):
         try:
             session.load(filepath, names)
             self._reset()
-            self._add_arrays(session)
-            self._listwidget.setCurrentRow(0)
+            self._push_data(session)
```
Comment (gdementen) [inline larray_editor/editor.py]: AFAICT, _add_arrays is not longer called. If this is indeed the case, it should be removed.

File: doc/source/changes/version_0_31.rst.inc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Miscellaneous improvements
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*
+
+
+Fixes
+^^^^^
+
+* fixed exception raised when loading a session from a file
```
Comment (gdementen) [inline doc/source/changes/version_0_31.rst.inc]: it only failed if the session contained non-array objects, right?

Comment (alixdamman) [reply doc/source/changes/version_0_31.rst.inc]: Yes, the changelog is indeed incomplete. Sorry.

File: larray_editor/editor.py (hunk 1 above)
Comment (alixdamman) [reply larray_editor/editor.py]: I don't see the reason to call "self._listwidget.setCurrentRow(0)" in self.new() ?
I checked and no exception was raised.

**Pull request merged**